client/src/assets/scss/_new.scss

```diff
@@ -4837,4 +4836,48 @@ div.stock-management {
       transform: rotate(180deg);
     }
   }
+
+  .label.logged-in span {
+    color: $color-primary;
+    font-size: 25px;
+    margin-left: 10px;
+  }
+
+  .ui.full.segment {
+
+    .form-input.batch-number,
+    .form-input.correction-reason {
+      background: $color-white;
+      border: 2px solid $color-light-gray;
+      color: $color-text;
+      padding: 5px;
+      text-align: center;
+      width: 50%;
+    }
+
+    .form-input.quantity {
+      border: 2px solid $color-light-gray;
+      margin-bottom: 30px;
+      width: 50%;
+    }
+
+    .form-input.quantity input {
+      background: $color-white;
+      border: none;
+      color: $color-text;
+      height: 100%;
+    }
+
+    .form-input.textarea {
+      background: $color-white;
+      border: 2px solid $color-light-gray;
+      color: $color-text;
+      padding: 10px;
+    }
+  }
+
+  // .overlay .ui.modal.diagnosis-popup {
+  //   margin-left: -250px;
+  //   width: 500px;
+  // }
```

Comment:
```suggestion
```

perhaps

Reply:
This is removed at follow up PR (#664), so no need to handle it here.